config/pipeline.yaml

```diff
@@ -920,6 +920,26 @@ jobs:
       job_timeout: 10
     kcidb_test_suite: kselftest.cpufreq
 
+  kselftest-cpufreq-suspend:
+    template: kselftest.jinja2
```

Comment:
This job template no longer exists. It was replaced by [`generic.jinja2`](https://github.com/kernelci/kernelci-pipeline/blob/main/config/runtime/generic.jinja2) - see [the example](https://github.com/kernelci/kernelci-pipeline/blob/main/doc/developer-documentation.md#an-example-of-enabling-a-new-job) for more details.

Reply:
@pawiecz this is a bit confusing. I see quite different threads here https://github.com/kernelci/kernelci-core/pull/2626
too.
We just want to add parameter support and for that we needed kselftest.jinja2 ( see https://github.com/Linaro/test-definitions/pull/511 )

Reply:
You have to rebase as there have been significant changes, and make it similar to other kselftest job definitions: [link](https://github.com/kernelci/kernelci-pipeline/blob/main/config/pipeline.yaml#L898)

Reply:
@a-wai thanks, I just forced pushed the changes after rebasing my branch